Index.html

```diff
@@ -6,6 +6,6 @@
     <title>CSD120-Erika Taege</title>
 </head>
 <body>
-    "Hello World!"
+    Hello World!
```

Comment:
When I give you instructions to insert text into your document and that text is in quotes, don't include the quotes